### PR TITLE
Remove duplicate functions in DLMFtex2Wiki.py

### PR DESCRIPTION
**Pull request by pwang00**
Import tex2Wiki from DLMFtex2Wiki

### CODE REVIEW

**Comment by physikerwelt** [comment]
This will address the issue #41

File: Azeem/src/DLMFtex2Wiki.py [hunk 1]
```diff
@@ -3,200 +3,15 @@
...
-    symList.append(symbol)
-    symList+=getSym(symbol)
-    return(symList)
-def unmodLabel(label):
-    label=label.replace(".0",".")
-    label=label.replace(":0",":")
-    return(label)
-def secLabel(label):
-    return(label.replace("\'\'",""))
-def modLabel(label):
```
Comment (physikerwelt) [inline Azeem/src/DLMFtex2Wiki.py]: mod Label is different

**Comment by pwang00** [comment]
the iterations variable wasn't updated for tex2Wiki, fixed that just now.

**Comment by pwang00** [comment]
@HowardCohl Duplicate functions are: isnumber, getString, getG, getEq, getEqP, getSym, unmodLabel, secLabel

**Pull request closed without merging**